simplq/src/components/pages/Admin/PauseQueue.jsx

```diff
@@ -1,13 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
 import PauseIcon from '@material-ui/icons/Pause';
+import PlayArrowIcon from '@material-ui/icons/PlayArrow';
 import SidePanelItem from 'components/common/SidePanel/SidePanelItem';
+import { usePauseQueue } from 'store/asyncActions/pauseQueue';
+// import { selectQueueDetails } from 'store/selectedQueue';
+
+export default ({ queueId }) => {
+  const dispatch = useDispatch();
+  const pauseQueue = usePauseQueue();
+  // const queueDetails = useSelector(selectQueueDetails);
+
+  const [description, setDescription] = useState('Temporarily stop people from joining');
+  const [isActive, setIsActive] = useState(false);
+
+  const onPanelClick = (chosenStatus) => {
+    dispatch(pauseQueue({ queueId, status: chosenStatus }));
+  };
+
+  // TODO: clean up this implementation
+  const getPausedOptions = () => {
+    if (!isActive) {
+      setDescription('Start allowing people to join the queue');
+      onPanelClick('PAUSED');
+      setIsActive(true);
+    } else {
+      setDescription('Temporarily stop people from joining');
+      onPanelClick('ACTIVE');
+      setIsActive(false);
+    }
+  };
 
-export default () => {
   return (
     <SidePanelItem
-      Icon={PauseIcon}
-      title="Pause Queue"
-      description="Temporarily stop people from joining"
+      Icon={isActive ? PlayArrowIcon : PauseIcon}
+      title={isActive ? 'Resume Queue' : 'Pause Queue'}
+      description={description}
+      onClick={getPausedOptions}
+      style={!isActive ? 'side-panel-item' : 'side-panel-item-alt'}
     />
```

Comment:
Let's call the style classes 'side-panel-item-active'  and 'side-panel-item-inactive' to make them clear